torch/_inductor/remote_cache.py

```diff
@@ -1,6 +1,6 @@
-# mypy: allow-untyped-defs
 import os
 from abc import abstractmethod
+from typing import Optional
 
 
 class RemoteCacheBackend:
```

Comment:
Shouldn't this be inherit from ABC or else the abstractmethod decorators are useless?

Reply:
Without ABC you don't get the runtime checking but mypy will still complain.

Reply:
Interesting, didn't know if mypy would still complain.